geomesa-kafka/geomesa-kafka-datastore/geomesa-kafka-10-datastore/src/main/scala/org/locationtech/geomesa/kafka10/LiveKafkaConsumerFeatureSource.scala

```diff
@@ -33,7 +33,8 @@ class LiveKafkaConsumerFeatureSource(e: ContentEntry,
                                      cleanUpCache: Boolean,
                                      useCQCache: Boolean,
                                      q: Query,
-                                     monitor: Boolean)
+                                     monitor: Boolean,
+                                     cleanUpCachePeriod: Long = 1000L)
```

Comment:
I think the default should be 10s or 30s. the 1s default is def too low bc it kills the machines if you want a big kafka datastore :)